Crash when exporting collection with time data while using SchedV2

###### Reproduction Steps

1. Open AnkiDroid with two decks and several sub deck. Enable Scheduler V2
2. Try to export a single deck with time data
3. Try to export whole collection without time data
4. Try to export whole collection while including time data


###### Expected Result
Trying to export a single deck shows a warning, that exporting single decks with time data is only supported with SchedV1.
When exporting the complete collection with time data, no such warning is shown, so I expect it to be able to export the collection


###### Actual Result
 Using SchedV2 the collection can be exported without the time data.

When the time data is selected for export, AnkiDroid crashes without further notice when trying to export the collection.

###### Debug info
AnkiDroid Version = 2.10beta3

Android Version = 10

ACRA UUID = 521087f7-500c-4737-9177-e052dde31e0e

###### Research
[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/521087f7-500c-4737-9177-e052dde31e0e

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$4.done(AsyncTask.java:399)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:289)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.lang.Thread.run(Thread.java:919)
Caused by: java.lang.NullPointerException: Attempt to read from field 'java.lang.Object android.util.Pair.first' on a null object reference
at com.ichi2.libanki.Note._fieldOrd(Note.java:1)
at com.ichi2.libanki.Note.setItem(Note.java:1)
at com.ichi2.libanki.AnkiPackageExporter._addDummyCollection(AnkiPackageExporter.java:6)
at com.ichi2.libanki.AnkiPackageExporter.exportVerbatim(AnkiPackageExporter.java:5)
at com.ichi2.libanki.AnkiPackageExporter.exportInto(AnkiPackageExporter.java:4)
at com.ichi2.async.CollectionTask.doInBackgroundExportApkg(CollectionTask.java:12)
at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:35)
at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:1)
at android.os.AsyncTask$3.call(AsyncTask.java:378)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
... 4 more
```

How amusing - Google runs a bunch of devices in a crawler mode against the app on every release, and normally there are no crashes these days but wouldn't you know the 2.10beta4 and 2.11alpha1 builds both had a single crash, right in this code:

```
05-10 16:22:45.995: E/AndroidRuntime(18919): java.lang.RuntimeException: An error occurred while executing doInBackground()
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at android.os.AsyncTask$4.done(AsyncTask.java:399)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:289)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.lang.Thread.run(Thread.java:919)
05-10 16:22:45.995: E/AndroidRuntime(18919): Caused by: java.lang.NullPointerException: Attempt to read from field 'java.lang.Object android.util.Pair.first' on a null object reference
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.libanki.Note._fieldOrd(Note.java:1)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.libanki.Note.setItem(Note.java:1)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.libanki.AnkiPackageExporter._addDummyCollection(AnkiPackageExporter.java:6)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.libanki.AnkiPackageExporter.exportVerbatim(AnkiPackageExporter.java:5)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.libanki.AnkiPackageExporter.exportInto(AnkiPackageExporter.java:5)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.async.CollectionTask.doInBackgroundExportApkg(CollectionTask.java:12)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:35)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at com.ichi2.async.CollectionTask.doInBackground(CollectionTask.java:1)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at android.os.AsyncTask$3.call(AsyncTask.java:378)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
05-10 16:22:45.995: E/AndroidRuntime(18919): 	... 4 more
```

That doesn't increase priority or add information (unless you didn't know we had an auto-crawl on each release, that's interesting info) but is an amazing coincidence

--- a/AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java
+++ b/AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.java
@@ -22,7 +22,6 @@ import com.ichi2.anki.CollectionHelper;
 import com.ichi2.anki.R;
 import com.ichi2.anki.exception.ImportExportException;
 import com.ichi2.compat.CompatHelper;
-
 import com.ichi2.utils.JSONArray;
 import com.ichi2.utils.JSONException;
 import com.ichi2.utils.JSONObject;
@@ -444,7 +443,9 @@ public final class AnkiPackageExporter extends AnkiExporter {
         f.delete();
         Collection c = Storage.Collection(context, path);
         Note n = c.newNote();
-        n.setItem("Front", context.getString(R.string.export_v2_dummy_note));
+        //The created dummy collection only contains the StdModels.
+        //The field names for those are localised during creation, so we need to consider that when creating dummy note
+        n.setItem(context.getString(R.string.front_field_name), context.getString(R.string.export_v2_dummy_note));
         c.addNote(n);
         c.save();
         c.close();

